scripts/debian/reversion-helper.sh

```diff
@@ -68,6 +68,12 @@ function reversion() {
     rm -rf "${__new_deb}"
     rm -rf "${__parent_dir}"/"${__new_deb}.deb"
 
+    if [[ ! -f "${__deb}_${__source_version}.deb" ]]; then
```

Comment:
Added more verbose listing of directory on missing debian. This was helpful when debugging